firmware/quintuna/RSM/src/io/io_rPump.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <stdbool.h>
+
+#ifdef TARGET_EMBEDDED
+#include "hw_i2cs.h"
+typedef struct
+{
+    const I2cDevice *src;
+} Pump;
+#else
+#include "app_utils.h"
+EMPTY_STRUCT(Pump);
+#endif
```

Comment:
you don't need to make an object if there's only one global object (or a very small amount of global objects)

Reply:
just use the functions to directly manipulate the given i2c device